docs/matrix.md

```diff
@@ -25,8 +26,60 @@ Tekton.
 ## Configuring a Matrix
 
 A `Matrix` supports the following features:
+* [Parameters](#parameters)
 * [Context Variables](#context-variables)
 
+### Parameters
+
+The `Matrix` will take `Parameters` of type `Array` only, which will be supplied to the
```

Comment:
Total nit: users would only ever type `"array"` and `"string"` in their yaml I think.  Suggest using these in place of `Array` and `String`.

Reply:
makes sense, updated them to `"array"` and `"string"`, thanks!